### ISSUE TITLE
Allow a suggested citation to be defined

### ISSUE THREAD

**Issue by phargogh** [natcap/data.naturalcapitalproject.stanford.edu#32]
The UK Natural History Museum has a suggested citation on their pages ([example](https://data.nhm.ac.uk/dataset/the-2016-release-of-the-predicts-database)), which would be a nice feature to add.

**Comment by phargogh** [comment]
I wonder if https://github.com/datopian/ckanext-doi is related?

**Comment by phargogh** [comment]
So it turns out that `ckanext-doi` _is_ related, but there's an administratively separate issue of using Stanford's DataCite subscription to pay for it, which will cost on the order of $1000/year for access.

While looking into this , I also started looking into `ckanext-attribution`, but couldn't immediately get the extension to compile due to issues with `spacy`.  It isn't clear if `ckanext-attribution` is strictly needed, and there are likely other ways to represent the information we need.  https://github.com/natcap/data.naturalcapitalproject.stanford.edu/issues/36 for that plugin.

At this time, the clearest, simplest path forwards appears to be through adding a custom field in our custom NatCap plugin ([CKAN API docs](https://docs.ckan.org/en/2.9/extensions/adding-custom-fields.html)), so I think that's what I'll do next.

**Comment by phargogh** [comment]
It turns out this can be done really easily with [`ckanext-scheming`](https://github.com/ckan/ckanext-scheming).

**Comment by phargogh** [comment]
Suggested citation field is live!

Next task: figure out how to edit the suggested citation via the API

**Comment by phargogh** [comment]
We have now figured out how to add a suggested citation via the api, so we this issue is now on hold until we figure out the MCF side of things.  Waiting on https://github.com/natcap/geometamaker/issues/17

**Comment by phargogh** [comment]
OK well, it turns out that I haven't yet figured out how to actually set the attribute yet when the attribute is defined in `ckanext-scheming`.  And the docs aren't particularly helpful on this front either.  Still investigating.

**Comment by phargogh** [comment]
It turns out that we implemented this elsewhere: https://github.com/natcap/data.naturalcapitalproject.stanford.edu/issues/76
